InvenTree/order/api.py

```diff
@@ -1304,7 +1304,7 @@ def download_queryset(self, queryset, export_format):
 
     filter_backends = [
         rest_filters.DjangoFilterBackend,
-        filters.SearchFilter,
+        InvenTreeSearchFilter,
         filters.OrderingFilter,
     ]
```

Comment:
Suggestion: Seems like this block is used often - can de deduplicate that?

Reply:
Yeah, this would be a good candidate for a refactoring